viewer/add-heading-anchors.js

```diff
@@ -37,7 +40,7 @@ var addHeadingAnchors = {
       class: "headerLink"
     };
 
-    icon.setAttribute("class", "icon-share");
+    icon.setAttribute("class", "icon-fixed-width icon-share");
```

Comment:
This was apparently still in policystat, so i must have neglected to ever PR this on this side.